services/product-service/src/functions/getProductsList/handler.ts

```diff
@@ -1,15 +1,30 @@
 import { APIGatewayProxyResult } from "aws-lambda/trigger/api-gateway-proxy";
 import { middyfy } from "@libs/lambda";
 import { type ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
-import products from "../../products.json";
-
-const getProductsList: ValidatedEventAPIGatewayProxyEvent<
-  APIGatewayProxyResult
-> = async () => {
-  return {
-    statusCode: 200,
-    body: JSON.stringify(products),
+import { IDatabaseService } from "src/db-service";
+import { okResponse } from "src/responses";
+import { databaseService } from "src/dependencies/dependencies";
+
+const getProductsList =
+  (
+    databaseService: IDatabaseService
+  ): ValidatedEventAPIGatewayProxyEvent<APIGatewayProxyResult> =>
+  async (event) => {
+    console.log(event);
+    const productDocuments = await databaseService.scanProducts();
```

Comment:
db error 500 is not handled (I also probably missed it). You could put it from this line in a try-catch block like in createProduct. Additionally, you could add 500 responses in swagger.